src/GamePool.jl

```diff
@@ -154,7 +143,11 @@ for the definition of entropy in information theory.
 """
 function entropy2(counts::AbstractVector{<:Real})
     countsum = sum(counts)
-    return -sum(x -> xlogx(x / countsum), counts) / log(2)
+    return -sum(counts) do k
+        x = k / countsum
+        xlogx = x * log(x)
+        iszero(x) ? zero(xlogx) : xlogx
```

Comment:
doesn't matter much because it's fast, but with the conditional branch here (instead of factored out in a different `xlogx` function), I would haven't have computed the `log` unless x != 0

Reply:
I would have done that too but I checked the code in `LogExpFunctions.jl` and it was in this form, presumably so that `zero(xlogx)` is the correct type.